modules/convert.sh

```diff
@@ -0,0 +1,30 @@
+function convert {
+  checkargn $# 2
+  inputFile=$1
+  outputFile=$2
+  if [[ "$inputFile" != "" ]] && [[ "$outputFile" != "" ]]; then
+    ffmpeg -i $inputFile $outputFile -hide_banner
+    status=$?
+    if [ "$status" == 0 ]; then
+      echo "$inputFile has been successfully converted to $outputFile"
+    else
+      echo "conversion unsuccessful"
+    fi
+  elif [ $inputFile == "" ]; then
+    echo "Error: no input file"
+  elif [ $outputFile == "" ]; then
+    echo "Error: no output file"
+  else
```

Comment:
1. The if statements need double quotes around the variables:
`elif [ "$inputFile" == "" ]; then`.

1. Maybe include an `exit 1` after the error echos.